packages/editor/src/managers/clipboard/paste-manager.ts

```diff
@@ -193,6 +194,15 @@ export class PasteManager {
         ) {
           parent &&
             this._addBlocks(blocks.slice(0), parent, index, addBlockIds);
+          const lastBlockModel = this._editor.page.getBlockById(lastBlock.id);
+          if (
```

Comment:
Please add some comments for help understanding this complex `if`. Example:

``` ts
// On pasting image, replace the last focused paragraph instead of appending a new image block,
//  if this paragraph is empty.
```

This should apply to future code style if you are writing `if` with more than 3 expressions joined together.

Reply:
Ok, Thank you for your advice.